pkgs/development/python-modules/python-vagrant/default.nix

```
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, vagrant, fetchPypi, lib }:
```

```suggestion
{ lib, buildPythonPackage, vagrant, fetchPypi }:
```
The same should be done for the other packages.